uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-legend.component.js

```diff
@@ -46,6 +47,17 @@ define([
       initWatchers();
     }());
 
+    /**
+     * Checks if the given absence type has a defined id or not.
+     * Useful for filtering absence types with defined ids.
```

Comment:
Sorry, this comment `Useful for filtering absence types with defined ids.` is a bit vague. It should not belong the func desc.